src/catalog/src/builtin.rs

```diff
@@ -7032,6 +7032,40 @@ pub static MZ_CLUSTER_REPLICA_HISTORY: LazyLock<BuiltinView> = LazyLock::new(||
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_CLUSTER_REPLICA_NAME_HISTORY: LazyLock<BuiltinView> = LazyLock::new(|| BuiltinView {
```

Comment:
This is useful for the cluster utilization graphs in the Console since we want to see the name of it at any point in time.